styles.css

```diff
@@ -0,0 +1,284 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,400;0,700;1,400;1,700&display=swap');
+
+:root {
+  --primary-text: #000;
+  --secondary-text: #666;
+  --primary-bg: #fff;
+  --secondary-bg: #f7f9fa;
+  --tertiary-bg: #e0f3ff;
+  --warning-color: #ff1f13;
+  --warning-dark: #ca1910;
+  --warning-light: #ff4b42;
+  --primary-color: #39a2f2;
+  --dark-color: #1a85d6;
+  --light-color: #6cbaf7;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Merriweather', serif;
+  margin: 0;
+}
+
+/* Buttons */
+.btn {
+  font-weight: 700;
+  border-radius: 25px;
+  font-family: inherit;
+  cursor: pointer;
+}
+
+.btn-new {
+  color: white;
+  background: var(--primary-color);
+  border: none;
+  padding: 1rem 1.2rem;
+  outline: none;
+  cursor: pointer;
+}
+
+.btn-new:hover,
+.btn-new:focus {
+  outline: none;
+  background: var(--dark-color);
+}
+
+.btn-new:active {
+  background: var(--light-color);
+}
+
+.btn-delete {
+  color: var(--warning-color);
+  background: var(--primary-bg);
+  padding: 0.8rem 2rem;
+  border: 3px solid var(--warning-color);
+}
+
+.btn-delete:hover,
+.btn-delete:focus {
+  outline: none;
+  background: var(--warning-dark);
+  border: 3px solid var(--warning-dark);
+  color: white;
+}
+
+.btn-delete:active {
+  background: var(--warning-light);
+  border: 3px solid var(--warning-light);
+}
+
+.btn-save {
+  border: 3px solid var(--primary-color);
+  padding: 0.8rem 2rem;
+  color: var(--primary-color);
+  background-color: var(--primary-bg);
+}
+
+.btn-save:hover,
+.btn-save:focus {
+  border: 3px solid var(--dark-color);
+  color: white;
+}
+
+.btn-save:active {
+  border: 3px solid var(--light-color);
+  color: white;
+}
+
+.absolute {
+  position: absolute;
+  top: 0.5rem;
+  right: 0rem;
+  padding-bottom: 1rem;
+}
+
+.marginTop {
+  margin-top: 1rem;
+}
+
+/* Toggle */
+
+input[type="checkbox"] {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+label[for="dark"] {
+  display: inline-block;
+  height: 34px;
+  width: 60px;
+  background-color: var(--tertiary-bg);
+  border-radius: 34px;
+  position: relative;
+  flex: 0 0 auto;
+  margin-bottom: 1rem;
+}
+
+.slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  transition: .4s;
+}
+
+.slider:before {
+  position: absolute;
+  content: "";
+  height: 26px;
+  width: 26px;
+  left: 4px;
+  bottom: 4px;
+  background: var(--primary-color);
+  transition: .4s;
+  border-radius: 50%;
+}
+
+input:checked+.slider:before {
+  transform: translateX(26px);
```

Comment:
remember you can create locally scoped variables in any class. So your `.slider` could have `--slider-thumb-width: 26px` and you can use this variable instead of duplicating 26px everywhere.